modules/gapi/test/streaming/gapi_streaming_tests.cpp

```diff
@@ -35,7 +35,6 @@ void initTestDataPath()
         // Since G-API has no own test data (yet), it is taken from the common space
         const char* testDataPath = getenv("OPENCV_TEST_DATA_PATH");
```

Comment:
> OPENCV_TEST_DATA_PATH

Do we really need that?
It is already handled by `ts` module (OpenCV testing framework)

Reply:
I don't know. Can I get an example or a file where this is used?

Reply:
https://github.com/opencv/opencv/wiki/QA_in_OpenCV

Reply:
I looked at the ts.hpp. And `findDataFile`(call `findData`) do this step. But i don't understand, how i can to get around `addDataSearchPath(...)`. I can remove `initTestDataPath` in this case.